datadog/table_datadog_permission.go

```diff
@@ -23,7 +23,7 @@ func tableDatadogPermission(ctx context.Context) *plugin.Table {
 			{Name: "group_name", Type: proto.ColumnType_STRING, Transform: transform.FromField("Attributes.GroupName"), Description: "Name of the permission group."},
 			{Name: "created_at", Type: proto.ColumnType_TIMESTAMP, Transform: transform.FromField("Attributes.Created"), Description: "Creation time of the permission."},
 
-			// Other columns
+			// Other useful columns
```

Comment:
Most other tables had `Other useful columns`, so we should pick one set of wording and be consistent